### PR TITLE
[Improve] Translate comments into English for FlinkAppState.java

### LINKED ISSUE #2621: [Improve] Translate comments into English.
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-streampark/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

 Translate comments into English for FlinkAppState.

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by SoberChina**
## What changes were proposed in this pull request

Issue Number: close #2621

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): no

### CODE REVIEW

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/enums/FlinkAppState.java [hunk 1]
```diff
@@ -68,13 +68,13 @@ public enum FlinkAppState implements Serializable {
...
 
-  /** 射影中... */
+  /** Mapping */
   MAPPING(14),
 
-  /** 其他不关心的状态... */
+  /** Other statuses */
   OTHER(15),
 
   /** has rollback */
```
Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/enums/FlinkAppState.java]:
```suggestion
  /** Has rollback */
```

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/enums/FlinkAppState.java
Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/enums/FlinkAppState.java:1]: `added new job to database` -> `Added new job to database` 
`job SUCCEEDED on yarn`->`Job SUCCEEDED on yarn`
`has killed in Yarn` -> `Has killed in Yarn`

**Review by RocMarshal — COMMENTED** [review]
Hi, @SoberChina Thanks for the contribution.
I left a few minor comments.
PTAL in your free time~

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/enums/FlinkAppState.java
Comment (SoberChina) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/enums/FlinkAppState.java:1]: @RocMarshal  Thank you very much for your reminder.Yes, the comments should be in one sentence, and handwritten letters should be capitalized.

**Review by RocMarshal — APPROVED** [review]
LGTM

**Review by wuchunfu — APPROVED** [review]
LGTM

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**